src/Surfnet/ServiceProviderDashboard/Application/ViewObject/Service.php

```diff
@@ -53,14 +58,23 @@ class Service
      * @param bool $privacyQuestionsEnabled
      * @param EntityList $entityList
      * @param RouterInterface $router
+     * @param bool $productionEntitiesEnabled
      */
-    public function __construct($id, $name, $privacyQuestionsEnabled, EntityList $entityList, RouterInterface $router)
-    {
+    public function __construct(
+        $id,
+        $name,
+        $privacyQuestionsEnabled,
+        EntityList $entityList,
+        RouterInterface $router,
+        $productionEntitiesEnabled = false
+    ) {
```

Comment:
You could have added type definitions, but this is good enough for rock and roll!

Reply:
while at it, good point!